src/ruler.js

```diff
@@ -70,8 +88,8 @@ export class DragRulerRuler extends Ruler {
 	}
 
 	// The functions below aren't present in the orignal Ruler class and are added by Drag Ruler
-	dragRulerAddWaypoint(point, snap=true) {
-		if (snap) {
+	dragRulerAddWaypoint(point, options={snap: true}) {
```

Comment:
This will fail to set `snap: true` as default if options without `snap` in them are passed. Better use an empty object as default and then add `options.snap = options.snap ?? true` to the code below to set that value if it's unset